### PR TITLE
chore: add dependabot configuration (#265)

### PR DESCRIPTION
**Pull request by jhoermann**
See [Configuring Dependabot version updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates) for more info.

### CODE REVIEW

**Comment by jhoermann** [comment]
Added https://github.com/fi3ework/vite-plugin-checker/pull/311/commits/f6201867a936ebacf02db98789790670003660be to also fix #264 @fi3ework.

**Comment by jhoermann** [comment]
Github has added `directories` now: https://github.blog/changelog/2024-04-29-dependabot-multi-directory-configuration-public-beta-now-available/. So I'll change this PR accordingly.

**Comment by jhoermann** [comment]
I added the `directories` option now so that it'll automatically check all `package.json`s of this repo. @fi3ework if you want to have a look, it can also be reduced to the main directories (e.g. exclude the playground ones).

**Comment by fi3ework** [comment]
Thanks you! It's seems general good for now, we could change if needed in the future.

**Review by fi3ework — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    versioning-strategy: 'increase'
+    directories:
+      - '**/*'
+    schedule:
+      interval: 'daily'
```
Comment (fi3ework) [inline .github/dependabot.yml:8]:
```suggestion
      interval: 'weekly'
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    labels:
+      - 'dependencies'
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```
Comment (fi3ework) [inline .github/dependabot.yml:14]: weekly
```suggestion
      interval: 'weekly'
```

**Review by fi3ework — CHANGES_REQUESTED** [review]
Setting to `weekly` fine will be reasonable, `daily` is somehow too noisy.

**Comment by jhoermann** [comment]
Thanks for the merge @fi3ework! Nice to see some action again on this repo. Adding the `dependencies` label would make sense now as the dependabot tries to apply it and can't find it.

**Pull request merged**